webapp/templates/jury/jury_macros.twig

```diff
@@ -71,85 +71,100 @@
     </style>
 {% endmacro %}
 
-{% macro table(data, fields, num_actions, options) %}
+{% macro table(data, fields, num_actions, options, delete_list_endpoint, multidelete=true) %}
```

Comment:
Maybe we make `multidelete` false by dfault for now? Or even better: make `delete_list_endpoint` null by default, but if it is not `null`, we know we have multi delete?